packages/next/bin/next.ts

```diff
@@ -15,13 +15,12 @@ import arg from 'next/dist/compiled/arg/index.js'
 const defaultCommand = 'dev'
 export type cliCommand = (argv?: string[]) => void
 const commands: { [command: string]: () => Promise<cliCommand> } = {
-  build: async () => await import('../cli/next-build').then(i => i.nextBuild),
-  start: async () => await import('../cli/next-start').then(i => i.nextStart),
-  export: async () =>
-    await import('../cli/next-export').then(i => i.nextExport),
-  dev: async () => await import('../cli/next-dev').then(i => i.nextDev),
+  build: async () => import('../cli/next-build').then(i => i.nextBuild),
```

Comment:
This change seems unrelated to the pr 🤔

Reply:
I initially looked at implementing it here and was initially confused by the extra `await`s since they appeared un-needed but wasn't sure if it optimized transforms which it appears it doesn't so I removed them to prevent confusion